pkgs/test_api/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 0.2.20-nullsafety
```

Comment:
The previous version is not yet published, so you can merge this into that entry (0.2.19-nullsafety)

Reply:
Ok, will do.  Thanks.